editor/.eslintrc.js

```diff
@@ -161,19 +161,19 @@ module.exports = {
     '@typescript-eslint/explicit-member-accessibility': 'off',
     '@typescript-eslint/no-var-requires': 'off',
     '@typescript-eslint/no-shadow': ['error'],
-    '@typescript-eslint/strict-boolean-expressions': [
-      'error',
-      {
-        allowString: false,
-        allowNumber: false,
-        allowNullableObject: false,
-        allowNullableBoolean: true,
-        allowNullableString: false,
-        allowNullableNumber: false,
-        allowAny: false,
-        allowRuleToRunWithoutStrictNullChecksIKnowWhatIAmDoing: false,
-      },
-    ],
+    // '@typescript-eslint/strict-boolean-expressions': [
+    //   'error',
+    //   {
+    //     allowString: false,
+    //     allowNumber: false,
+    //     allowNullableObject: false,
+    //     allowNullableBoolean: true,
+    //     allowNullableString: false,
+    //     allowNullableNumber: false,
+    //     allowAny: false,
+    //     allowRuleToRunWithoutStrictNullChecksIKnowWhatIAmDoing: false,
+    //   },
+    // ],
```

Comment:
Bin it! This damn rule is such a pain when committing :( 

```suggestion
```

Reply:
I'm gonna chuck it on master